packages/pn-personafisica-webapp/src/navigation/SessionGuard.tsx

```diff
@@ -32,6 +33,8 @@ const INITIALIZATION_SEQUENCE = [
   INITIALIZATION_STEPS.SESSION_CHECK,
 ];
 
+const { DISABLE_INACTIVITY_HANDLER } = getConfiguration();
```

Comment:
Move this into SessionGuardRender

Reply:
So it will be used in condition https://github.com/pagopa/pn-frontend/pull/1224#discussion_r1642291692 ?